ci/build_container/build_recipes/yaml-cpp.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+# Using a pinned commit rather than release version to avoid taking the Boost
+# dependency present in last release.
```

Comment:
yuck. If we do this is there any other solution to this in terms of other libraries? Really don't want to depend on Boost. Is my understanding correct that for v2 protobuf will do YAML ingestion itself? Or do we still need a library?